config/main.py

```diff
@@ -149,11 +149,11 @@ def load_mgmt_config(filename):
     command = "ifconfig eth0 {} netmask {}".format(str(mgmt_conf.ip), str(mgmt_conf.netmask))
     run_command(command, display_cmd=True)
     command = "ip route add default via {} dev eth0 table default".format(gw_addr)
-    run_command(command, display_cmd=True)
+    run_command(command, display_cmd=True, ignore_error=True)
     command = "ip rule add from {} table default".format(str(mgmt_conf.ip))
-    run_command(command, display_cmd=True)
+    run_command(command, display_cmd=True, ignore_error=True)
     command = "[ -f /var/run/dhclient.eth0.pid ] && kill `cat /var/run/dhclient.eth0.pid` && rm -f /var/run/dhclient.eth0.pid"
-    run_command(command, display_cmd=True)
+    run_command(command, display_cmd=True. ignore_error=True)
```

Comment:
Looks like a typo, period instead of comma after first "True": ` display_cmd=True. ignore_error=True`